### ISSUE TITLE
Documentation wrong when adding Z-wave driver?

### ISSUE THREAD

**Issue by Inversion-NL** [athombv/node-homey-zwavedriver#27]
When you follow the manual to the letter it says:

> 
> **Installation**
> `cd /path/to/com.your.homeyapp/
> git submodule add https://github.com/athombv/node-homey-zwavedriver.git`
> 
> **Example**
> `const ZwaveDriver = require('homey-zwavedriver');`

But when you do that the Z-wave driver isn't downloaded to the node_modules sub dir.
So I went ahead and removed the module (git rm node-homey-zwavedriver) and changed to the node_modules directory and fired 'git submodule add https://github.com/athombv/node-homey-zwavedriver.git` once again.
Fired 'athom project --run' which results in:
![image](https://cloud.githubusercontent.com/assets/15921016/22246378/f5970c76-e234-11e6-9325-28244977587f.png)

So I went ahead and renamed the folder from 'node-homey-zwavedriver' to 'homey-zwavedriver' and now my driver works like a charm.

So, to sum up what has to be changed:

1. Change documentation: add a line to cd to node-modules directory
2. Change name of driver to node-homey-zwavedriver or rename repository on Git.
